crates/sui-graphql-rpc/src/types/move_package.rs

```diff
@@ -67,6 +67,8 @@ impl MovePackage {
                 .clone()
                 .into_iter()
                 .collect::<Vec<(String, Vec<u8>)>>();
+
+            // ok to unwrap because we know mod_map is not empty
```

Comment:
An empty module list should probably be an internal error, because no published package can have no modules.